<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SZDJZBDT0105" ignored="0" name="testng" tests="5" failures="1" timestamp="21 一月 2019 02:30:05 GMT" time="0.056" errors="0">
  <testcase name="test01" time="0.011" classname="demo3.ReRunFailedTestCaseDemo"/>
  <testcase name="test02" time="0.005" classname="demo3.ReRunFailedTestCaseDemo">
    <failure type="java.lang.AssertionError" message="expected [13] but found [11]">
      <![CDATA[java.lang.AssertionError: expected [13] but found [11]
at demo3.ReRunFailedTestCaseDemo.test02(ReRunFailedTestCaseDemo.java:13)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test02 -->
  <testcase name="test03" time="0.002" classname="demo3.ReRunFailedTestCaseDemo"/>
  <testcase name="test03" time="0.001" classname="demo3.ReRunFailedTestCaseDemo"/>
  <testcase name="test03" time="0.001" classname="demo3.ReRunFailedTestCaseDemo"/>
</testsuite> <!-- testng -->
